instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -262,18 +262,24 @@ def _instrument(
     url_filter: _UrlFilterT = None,
     request_hook: _RequestHookT = None,
     response_hook: _ResponseHookT = None,
+    trace_configs: [aiohttp.TraceConfig] = None,
```

Comment:
Is this `[aiohttp.TraceConfig]` valid? Shouldn't it be `Optional[aiohttp.TraceConfig]`?